app/validation.py

```diff
@@ -263,9 +273,10 @@ def _translate_json_schema_error(message):
             # A string that is too long
             return 'under_character_limit'
     if 'does not match' in message:
-        return 'under_{}_words'.format(_get_word_count(message))
-    if "is not of type 'number'" in message:
-        return 'not_a_number'
+        if key in ['priceMin', 'priceMax']:
+            return 'not_money_format'
```

Comment:
The prices are not the only numbers that get validated - there is also the 'serviceAvailability` percentage, which (as far as I know) we are keeping as a number.  So we should keep the `not_a_number` branch in here I think, as well as the new string money-format check for prices.